CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ All notable, unreleased changes to this project will be documented in this file.
 # 3.15.0 [Unreleased]
 
 ### Breaking changes
+Shipping methods can be removed by user after it has been assigned to a checkout `shipping_method_id` is no longer a  required field. `remove_shipping_method` has been added - #13068 by @FremahA
```

Comment:
Slightly redacted to include API names, also we don't have to mention internal functions in this case as here we want to document changes that are observable by API users.
```suggestion
Shipping methods can be removed by the user after it has been assigned to a checkout; `shippingMethodId` is now a nullable input in the `checkoutShippingMethodUpdate` mutation.  - #13068 by @FremahA
```

Reply:
Also it's not a breaking change, it can be moved to GraphQL API section.